v22.2/monitor-and-debug-changefeeds.md

```diff
@@ -45,12 +45,20 @@ Changefeed progress is exposed as a high-water timestamp that advances as the ch
     (1 row)
     ~~~
 
-- Setting up an alert on the `changefeed.max_behind_nanos` metric to track when a changefeed's high-water mark timestamp is at risk of falling behind the cluster's [garbage collection window](configure-replication-zones.html#replication-zone-variables). For more information, see [Monitoring and Alerting](monitoring-and-alerting.html#changefeed-is-experiencing-high-latency).
+- Using Prometheus and Alertmanager to track and alert on changefeed metrics. See the [Monitor CockroachDB with Prometheus](monitor-cockroachdb-with-prometheus.html) tutorial for steps to set up Prometheus. The following [Recommended changefeed metrics](#recommended-changefeed-metrics) section describes essential metrics to alert you when a changefeed encounters a retryable error, or enters a failed state.
 
 {{site.data.alerts.callout_info}}
 You can use the high-water timestamp to [start a new changefeed where another ended](create-changefeed.html#start-a-new-changefeed-where-another-ended).
 {{site.data.alerts.end}}
 
+### Recommended changefeed metrics
```

Comment:
supernit: `### Recommended changefeed metrics to track` otherwise we could be recommending these metrics for an award, or for parole.